mmv1/templates/terraform/examples/region_backend_service_ilb_custom_metrics.tf.tmpl

```diff
@@ -0,0 +1,52 @@
+resource "google_compute_network" "default" {
+  provider = google-beta
+  name                    = "{{index $.Vars "network_name"}}"
+}
+
+// Zonal NEG with GCE_VM_IP_PORT
+resource "google_compute_network_endpoint_group" "default" {
+  provider = google-beta
+  name                  = "{{index $.Vars "default_neg_name"}}"
+  network               = google_compute_network.default.id
+  default_port          = "90"
+  zone                  = "us-central1-a"
+  network_endpoint_type = "GCE_VM_IP_PORT"
+}
+
+resource "google_compute_region_backend_service" "{{$.PrimaryResourceId}}" {
+  provider = google-beta
+
+  region = "us-central1"
+  name = "{{index $.Vars "region_backend_service_name"}}"
+  health_checks = [google_compute_health_check.health_check.id]
+  load_balancing_scheme = "INTERNAL_MANAGED"
+  locality_lb_policy    = "WEIGHTED_ROUND_ROBIN"
+  custom_metrics {
+    name    = "orca.application_utilization"
+    # At least one metric should be not dry_run.
+    dry_run = false
+  }
+  backend {
+    group = google_compute_network_endpoint_group.default.id
+    balancing_mode = "CUSTOM_METRICS"
+    custom_metrics {
+      name    = "orca.cpu_utilization"
+      max_utilization = 0.9
+      dry_run = true
+    }
+    custom_metrics {
+      name    = "orca.named_metrics.foo"
+      max_utilization = 0.9
```

Comment:
Can you not set this value in one of these tests?

Reply:
Done